data/json/items/gun/22.json

```
@@ -407,5 +407,46 @@
     "weapon_category": [ "AUTOMATIC_PISTOLS" ],
     "pocket_data": [ { "magazine_well": "126 ml", "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "wp22mag" ] } ],
     "melee_damage": { "bash": 4 }
+  },
+  {
+    "id": "cp33",
+    "copy-from": "pistol_base",
+    "looks_like": "ar_pistol",
+    "type": "GUN",
+    "name": { "str": "high capacity .22 handgun" },
```

Ammotypes are automatically added to names by default
```suggestion
    "name": { "str": "high capacity handgun" },
```

---

Ah, thanks!